src/Router.tsx

```diff
@@ -27,6 +29,14 @@ const InnerRouter = () => {
           index: true,
           element: <IndexScreen />,
         },
+        {
+          path: "/team/:teamName",
+          element: <TeamHistory />,
+        },
+        {
+          path: "/matchup/:team1&:team2",
```

Comment:
I was envisioning this to be more like:
`/matchup/ars-mci`

Where `ars` and `mci` come from some constants file where each team has a 3 letter abbreviation. What do you think?

Reply:
sure then we have to map them to anther constant to query them from the BE, whereas right now we can just take the team1 and team2 from query params and pass them in easily..

Reply:
I dont think it's a lot of work to map to a constant, I can do them at the API level. Also avoids spaces and other special characters like this
<img width="335" alt="image" src="https://github.com/rahulr8/sports-betting-stats/assets/37151383/2866ba19-1f80-4422-a967-664ee1f788e8">